### DISCUSSION TITLE
Cables that include SFPs & Inventory

### DISCUSSION THREAD

**Discussion by jscooksey** [netbox-community/netbox#13964] [Q&A]
For cables that have the SFP modules as part of the cable such as: 
Cisco SFP-H10GB-CU1M 10GBase-CX1 SFP Module 1-Meter Twinax 10G Cable

The SFP modules show up as inventory on both devices they are connected to, but are not individual, they share a serial number.
So I can create two inventory items, and attach one to each device, but that isn't right
Cables dont get allocated serial numbers, fair enough.

Any better options to record this?

**Comment by mtinberg** [comment]
While I am not a Netbox developer, I can always chime in to say that perfect modeling can provide diminishing returns for the time and complexity needed. Inventory Items are more informational than representing config that needs to be pushed to devices, so is there a problem if you search by serial number and bring up both Interfaces, or need to de-duplicate reporting on counts? What's the cost of the dups vs the complexity of making the modeling more accurate in this case?


—

Mark Tinberg ***@***.***>

Division of Information Technology-Network Services

University of Wisconsin-Madison


________________________________
From: Justin Cooksey ***@***.***>
Sent: Tuesday, October 3, 2023 7:06 PM
To: netbox-community/netbox ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [netbox-community/netbox] Cables that include SFPs & Inventory (Discussion #13964)


For cables that have the SFP modules as part of the cable such as:
Cisco SFP-H10GB-CU1M 10GBase-CX1 SFP Module 1-Meter Twinax 10G Cable

The SFP modules show up as inventory on both devices they are connected to, but are not individual, they share a serial number.
So I can create two inventory items, and attach one to each device, but that isn't right
Cables dont get allocated serial numbers, fair enough.

Any better options to record this?

**Comment by jcollie** [comment]
> Cables dont get allocated serial numbers, fair enough.
> 
> Any better options to record this?

You can add a custom field to the Cable model called "serial number". It isn't quite as nice as a "real" field but it's there. I use quite a lot of custom fields to capture these sort of things.
